playkit/src/main/java/com/kaltura/playkit/player/TrackSelectionHelper.java

```diff
@@ -126,10 +126,11 @@
     }
 
     interface TracksErrorListener {
-
         void onTracksOverrideABRError(PKError pkError);
         void onUnsupportedVideoTracksError(PKError pkError);
-
+        void onUnsupportedAudioTracksError(PKError pkError);
+        void noSupportedAudioVideoTracksError(PKError pkError);
```

Comment:
lets make it same as all other errors on*...

Reply:
Done